src/methods/function.js

```diff
@@ -1184,13 +1184,7 @@ export function EvaluateStatements(
             } else {
               invariant(blockValue instanceof PossiblyNormalCompletion);
               invariant(res instanceof PossiblyNormalCompletion);
-              let e = realm.getCapturedEffects();
```

Comment:
Bunch of lines deleted here. Not needed?

Reply:
Apparently not. Just deleting them (in the absence of the other changes) does not make for detectable difference, but reading over the code it no longer made sense to me. A bit of red herring here, so feel free to ignore.